qa/workunits/rbd/rbd-nbd.sh

```diff
@@ -4,6 +4,7 @@ set -ex
 . $(dirname $0)/../../standalone/ceph-helpers.sh
 
 POOL=rbd
+ANOTHER_POOL=new_default_pool
```

Comment:
nit: probably it will be safer to randomize the name a little, e.g. `ANOTHER_POOL=new_default_pool$$`.

It is highly unlikely some one will step on this, on the other hand it is not difficult to be safer here.

Reply:
Done